src/compile/data/nonpositivenullfilter.ts

```diff
@@ -1,5 +1,5 @@
 import {ScaleType} from '../../scale';
```

Comment:
Maybe it's my own mistake, but I guess this should become `nonpositivefilter.ts`?